src/main/java/com/redislabs/WriteSource.java

```diff
@@ -41,7 +41,8 @@ public WriteSource(String name, String connector, String xmlDef, boolean writeTh
    
     KeysReader reader = new KeysReader().
         setPattern(getHashPrefix() + ":*").
-        setEventTypes(new String[] {"hset", "hmset", "hincrbyfloat", "hincrby", "hdel", "del", "change"});
+        //setEventTypes(new String[] {"hset", "hmset", "hincrbyfloat", "hincrby", "hdel", "del", "change"});
+        setEventTypes(new String[] {"hset", "hmset", "hincrbyfloat", "hincrby", "hdel", "del"});
```

Comment:
Why remove the `change`?

Reply:
Added an option to disable write on change event.